tests/queryset_pickle/tests.py

```diff
@@ -55,6 +55,18 @@ def test_multipleobjectsreturned_class(self):
         klass = Event.MultipleObjectsReturned
         self.assertIs(pickle.loads(pickle.dumps(klass)), klass)
 
+    def test_forward_relatedobjectcoesnotexist_class(self):
```

Comment:
Typo in both test names:

`relatedobjectcoesnotexist` -> `relatedobjectdoesnotexist`

Reply:
Thanks, just fixed it.